lib/zendesk_api/collection.rb

```diff
@@ -357,14 +357,13 @@ def intentional_obp_request?
     end
 
     def should_try_cbp?(path_query_link)
-      not_supported_endpoints = %w[show_many]
-      not_supported_endpoints.none? { |endpoint| path_query_link.end_with?(endpoint) }
+      @next_page.nil? && @options.delete(:attempt_cbp)
```

Comment:
It would be nice to explain why `@next_page.nil?`, in a smaller class this could be extracted to an intention revealing method, but maybe just a local var or a comment would work in this case.